genesis/src/genesis_accounts.rs

```diff
@@ -280,7 +281,19 @@ pub fn add_genesis_accounts(genesis_config: &mut GenesisConfig) -> u64 {
         &POOL_STAKER_INFOS,
         &UNLOCKS_BY_TENTHS_FOR_60_MONTHS,
         sol_to_lamports(1_000_000.0),
-    ) + add_validators(genesis_config, &VALIDATOR_INFOS)
+    ) + add_validators(genesis_config, &VALIDATOR_INFOS);
+
+    // "one thanks" gets 500_000_000SOL (total) - above distributions
+    create_and_add_stakes(
+        genesis_config,
+        &StakerInfo {
+            name: "one thanks",
+            staker: "3b7akieYUyCgz3Cwt5sTSErMWjg8NEygD6mbGjhGkduB",
+            sol: 500_000_000.0 - lamports_to_sol(issued_lamports),
```

Comment:
This could go negative.   Skip this `create_and_add_stakes()` entire if so?

Reply:
I have another solution, ignore faucet lamports

Reply:
Yes, ignore faucet lamports